### PR TITLE
LPS-177062 TR2 Automated Test | master

### PR DESCRIPTION
**Pull request by joelrocha68**
[LPS-177062](https://issues.liferay.com/browse/LPS-177062)

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:site-initializer-testray:nocompile

**Comment by joelrocha68** [comment]
ci:test:site-initializer-testray:nocompile

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:site-initializer-testray:nocompile

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase [hunk 1]
```diff
@@ -0,0 +1,227 @@
+@component-name = "portal-solutions"
+definition {
+
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase:3]: Tests are failing on CI.
Could you please try adding skip.portal.instance for these tests and we can run it again to see if that helps.

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+		}
+	}
+
+	@description = "Story ID: LPS-173530 | Test Summary: Verify if as an Administrator I should be able to filter case requirements by link"
+	@priority = 5
+	test AdminCanFilterCaseRequirementsByLink {
+		task ("When the user filter a case requirement by link") {
+			TestrayFilter.clickFilter();
+
+			Type(
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase:118]: macros please

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+		}
+
+		task ("Then can see the filters apply in the list by link") {
+			TestrayRequirements.verifyRequirement(
+				linkTitle = "The front page of the internet",
+				requirementComponent = "Component 1",
+				requirementSummary = "Requirement 1",
+				requirementTeam = "Team 1");
+
+			Testray.assertItemNotPresent(key_name = "Requirement 2");
```
Comment (nikki-pru) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase:133]: AssertElementNotPresent

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase (hunk 1 above)
Comment (joelrocha68) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase:3]: Done

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase (hunk 2 above)
Comment (joelrocha68) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase]: Done

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase (hunk 3 above)
Comment (joelrocha68) [reply modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayCaseRequirementsFilters.testcase]: Done

**Comment by joelrocha68** [comment]
ci:test:sf

**Pull request closed without merging**